src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractOverviewIndexWriter.java

```diff
@@ -62,35 +62,35 @@ public AbstractOverviewIndexWriter(HtmlConfiguration configuration,
      * summary at the top of the page and generate a link to the description,
      * which is added at the end of this page.
      *
-     * @param main the documentation tree to which the overview header will be added
+     * @param target the documentation to which the overview header will be added
      */
-    protected void addOverviewHeader(Content main) {
-        addConfigurationTitle(main);
-        addOverviewComment(main);
-        addOverviewTags(main);
+    protected void addOverviewHeader(Content target) {
+        addConfigurationTitle(target);
+        addOverviewComment(target);
+        addOverviewTags(target);
     }
 
     /**
      * Adds the overview comment as provided in the file specified by the
      * "-overview" option on the command line.
      *
-     * @param htmltree the documentation tree to which the overview comment will
+     * @param content the documentation to which the overview comment will
```

Comment:
If the parameter name was `target`, you could replace `documentation` with `content`.